Classes/KIFTestActor.m

```diff
@@ -169,6 +174,36 @@ + (void)setStepDelay:(NSTimeInterval)newStepDelay;
     KIFTestStepDelay = newStepDelay;
 }
 
++ (void)setEnableAutocorrect:(BOOL)enableAutocorrect;
+{
+    KIFAutocorrectEnabled = enableAutocorrect;
```

Comment:
I think there is a bit of a mixed paradigm here between having the static bool configuration and having the `KIFSetAutocorrect` methods, but only updated at `+setUp`. I think this might have been caused by our offline conversation where I'd pointed at putting this setting in here, but I thought you were going in a slightly different direction. Sorry if I caused confusion here.

Thoughts on having these just call directly through to `KIFSetAutocorrect(enableAutocorrect)`? I don't think we actually need the getter or the static BOOL for anything in that case. You could then make the `KIFTestCase` changes be done once in `+[KIFTestActor load]` setting it to `NO` to start, so that it doesn't stomp changes made by tests before they call `+[KIFTestCase setUp]` and it also doesn't require subclassing of `KIFTestCase` if that isn't desired.

I think the tests would stay exactly the same, except you could call `[super setUp]` before or after `setEnableAutocorrect:` and it wouldn't matter.